code/modules/mob/mob_defines.dm

```diff
@@ -106,6 +106,7 @@
 	var/candrop = 1
 	var/lastpuke = 0
 	var/unacidable = 0
+	var/nospells = 0 //Can't use spells. At all.
```

Comment:
So adding a proc and a mob-level var is better than just the var?
Or did you mean adding a new event to be invoked just for when someone selects a spell would be better?